static/app/views/replays/replays.tsx

```diff
@@ -92,47 +93,57 @@ class Replays extends React.Component<Props> {
   render() {
     const {organization} = this.props;
     return (
-      <PageFiltersContainer
-        showEnvironmentSelector={false}
-        resetParamsOnChange={['cursor']}
-      >
-        <PageContent>
-          <PageHeader>
-            <HeaderTitle>
-              <div>
-                {t('Replays')} <FeatureBadge type="alpha" />
-              </div>
-            </HeaderTitle>
-          </PageHeader>
-
-          <DiscoverQuery
-            eventView={this.getEventView()}
-            location={this.props.location}
-            orgSlug={organization.slug}
-          >
-            {data => {
-              return (
-                <React.Fragment>
-                  <PanelTable
-                    isLoading={data.isLoading}
-                    isEmpty={data.tableData?.data.length === 0}
-                    headers={[t('Session'), t('Timestamp')]}
-                  >
-                    {data.tableData
-                      ? this.renderTable(data.tableData.data as Replay[])
-                      : null}
-                  </PanelTable>
-                  <Pagination pageLinks={data.pageLinks} />
-                </React.Fragment>
-              );
-            }}
-          </DiscoverQuery>
-        </PageContent>
-      </PageFiltersContainer>
+      <React.Fragment>
+        <StyledPageHeader>
+          <HeaderTitle>
+            <div>
+              {t('Replays')} <FeatureBadge type="alpha" />
+            </div>
+          </HeaderTitle>
+        </StyledPageHeader>
+        <PageFiltersContainer hideGlobalHeader resetParamsOnChange={['cursor']}>
+          <StyledPageContent>
+            <DiscoverQuery
+              eventView={this.getEventView()}
+              location={this.props.location}
+              orgSlug={organization.slug}
+            >
+              {data => {
+                return (
+                  <React.Fragment>
+                    <ReplaysFilters />
+                    <PanelTable
+                      isLoading={data.isLoading}
+                      isEmpty={data.tableData?.data.length === 0}
+                      headers={[t('Session'), t('Timestamp')]}
+                    >
+                      {data.tableData
+                        ? this.renderTable(data.tableData.data as Replay[])
+                        : null}
+                    </PanelTable>
+                    <Pagination pageLinks={data.pageLinks} />
+                  </React.Fragment>
+                );
+              }}
+            </DiscoverQuery>
+          </StyledPageContent>
+        </PageFiltersContainer>
+      </React.Fragment>
     );
   }
 }
 
+const StyledPageHeader = styled(PageHeader)`
+  background-color: ${p => p.theme.surface100};
+  margin-top: 10px;
+  margin-left: 30px;
```

Comment:
Are these better represented by different `space()` values?

I'm new to what's available, but i'm surprised there isn't a component and we need to space things out like this manually.

Reply:
I'll look around some more and see what's there. I just put it together.

Reply:
Confirmed there's values in space.tsx. I'll use those.

Reply:
Adjusted